HerPublicWebsite.BusinessLogic/Models/Questionnaire.cs

```diff
@@ -115,7 +115,7 @@ public EpcRating EffectiveEpcBand
         }
     }
 
-    public bool EpcIsExpired => EpcDetails.ExpiryDate < DateTime.Now;
+    public bool EpcIsExpired => EpcDetails?.ExpiryDate == null ? false : EpcDetails.ExpiryDate < DateTime.Now;

```

Comment:
Rider (and I) prefer:
```suggestion
    public bool EpcIsExpired => EpcDetails?.ExpiryDate != null && EpcDetails.ExpiryDate < DateTime.Now;
```

Reply:
It suggested that to me too, but I don't like it. The date being null is a weird case so I like being explicit about how we're treating it.